op-node/rollup/derive/calldata_source.go

```diff
@@ -100,7 +104,7 @@ func (ds *DataSource) Next(ctx context.Context) (eth.Data, error) {
 // DataFromEVMTransactions filters all of the transactions and returns the calldata from transactions
 // that are sent to the batch inbox address from the batch sender address.
 // This will return an empty array if no valid transactions are found.
-func DataFromEVMTransactions(config *rollup.Config, txs types.Transactions, log log.Logger) []eth.Data {
```

Comment:
I don't think this function needs to take the entire `rollup.Config` anymore since it just uses `config.BatchInboxAddress`, could just pass in that instead of the entire config

Reply:
It also needs the chain ID to construct a signer with to get transaction sender information.